apps/web/src/forms/Q3SymptomCoughSeverity.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { PageProps } from '../types/PageProps';
+import { Question } from '../components/Question';
+import { RadioField } from '../components/RadioField';
+
+export const Q3SymptomCoughSeverity = ({ values, onChange }: PageProps) => {
+  const { t } = useTranslation();
+
+  const severity = values?.severityOfBreathing || 'None';
+  const noneChecked = severity === 'None' || severity === '';
+
+  return (
+    <div>
+      <div className='pb-5'>
+        <Question>{t('Q3.5-cough')}</Question>
+        <div className='text-base text-bcGray mb-2'>{t('Q3.5-cough-desc1')}</div>
+        <div className='text-base text-bcGray font-bold mb-2'>{t('Q3.5-cough-desc2')}</div>
+        <div className='text-base text-bcGray'>
+          <Trans i18nKey='Q3.5-cough-desc3'>
+            For more information on cough, see HealthLinkBC&apos;s information for
```

Comment:
Unfortunately, i18n supports only simple html tags like <b>, <strong> in translation.